packages/web-fragments/src/elements/reframed/script-execution.ts

```diff
@@ -85,6 +85,12 @@ export function executeInertScript(inertScript: HTMLScriptElement, iframeDocumen
 		return false;
 	}
 
+	// If the inert script has already been evaluated but later re-added to the DOM
```

Comment:
consider moving this up to the top of the function as this is a cheap check that can short-circuit the rest.